### ISSUE TITLE
Command Not Found - no linking with the .psm1 file

### ISSUE THREAD

**Issue by HighPriest** [farag2/Windows-10-Sophia-Script#88]
**Steps to reproduce**
------------------
1. Commented out a few functions from the .ps1 file
```PowerShell
Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process -Force
.\Sophia.ps1
```

**Screenshots**
![image](https://user-images.githubusercontent.com/58490197/98807266-afac8200-241a-11eb-88ce-99d55767a4ac.png)

**Desktop (please complete the following information):**
- Windows 10 Pro build: 19042.630

**Comment by farag2** [comment]
The preset file was updated too. Use the new one!
Check it: https://github.com/farag2/Windows-10-Sophia-Script/blob/d1168e2427631057948b4a2a3c03b2b2ac943268/Sophia/Sophia.ps1#L1268

**Comment by HighPriest** [comment]
No function is found.
I get errors for all of them.

**Comment by farag2** [comment]
Wow. Did you download a new archive from release page?

**Comment by HighPriest** [comment]
Yes, it's the latest

**Comment by farag2** [comment]
Did you reload the PS console before running? I mean seems PS "remembers" the previous commands. Just close and open again it.

**Comment by HighPriest** [comment]
Restarted the PowerShell console, no change.

**Comment by farag2** [comment]
WTF. Do you have an app for remote control? I prefer TeamViewer. I can check...

**Comment by farag2** [comment]
We can get in touch via Telegram if you want me to help. Cos if it's true, it's very odd.

**Comment by farag2** [comment]
So?

**Comment by HighPriest** [comment]
Sorry farag, had to take care of my laundry.

I don't have telegram and Windows has a very good Remote Help utility.  Discord maybe?

**Comment by farag2** [comment]
OK. Time?
```very good Remote Help utility```
Never used that. Yeah, I have a Discord.
sanctuary_s#6351

**Comment by HighPriest** [comment]
Sent friend request from Bunny

**Comment by HighPriest** [comment]
Just for clarity, the issue was fixed by updating to script v5.2
